data/scripts/actions/others/music_box.lua

```diff
@@ -0,0 +1,43 @@
+local config = {
+	["dragonling"] = {mountId = 31, tameMessage = "The wild dragonling will accompany you as a friend from now on.", sound = "FI?"},
+	["draptor"] = {mountId = 6, tameMessage = "The wild draptor will accompany you as a friend from now on.", sound = "Screeeeeeeeech"},
+	["enraged white deer"] = {mountId = 18, tameMessage = "The wild deer will accompany you as a friend from now on.", sound = "*bell*"},
+	["ironblight"] = {mountId = 29, tameMessage = "The ironblight will accompany you as a friend from now on.", sound = "Plinngggg"},
+	["magma crawler"] = {mountId = 30, tameMessage = "The magma crawler will accompany you as a friend from now on.", sound = "ZzzZzzZzzzZz"},
+	["midnight panther"] = {mountId = 5, tameMessage = "The wild panther will accompany you as a friend from now on.", sound = "Purrrrrrr"},
+	["wailing widow"] = {mountId = 1, tameMessage = "You have tamed the wailing widow.", sound = "Sssssssss"},
+	["wild horse"] = {mountId = 17, tameMessage = "The horse accepts you as its new master.", sound = "*snort*"},
+	["panda"] = {mountId = 19, tameMessage = "The panda will accompany you as a friend from now on.", sound = "Rrrrr... Grrrr"}
```

Comment:
Please make this multiline to overflow less, like:
```lua
["panda"] = {
  mountId = 19,
  tameMessage = "The panda will accompany you as a friend from now on.",
  sound = "Rrrrr... Grrrr",
},
```

Reply:
yes sir